meetings/2023-06-07.md

```diff
@@ -54,24 +69,71 @@ URL: https://www.w3.org/2023/09/TPAC/
+* HS: working on a project called Solid Wallet. The idea is to have a ??? which would be secured. Th way it works is you can have your keys on your pod protected so only the wallet can access them, keep them in a safe place ans sign requests on your behalf. It could also be a proxy on your pod that does that.  
+
 
 ### Compatibility with existing implementations
 URL: https://github.com/solid/solid-wg-charter/issues/31
 
+* SC: Any objections to keeping the paragraph in?
+* SC: No objections.
+
+ACTION: SC to close the issue and reflect group consensus. 
 
 ### Revise scope and out-of-scope
-* SC: Resolves https://github.com/solid/solid-wg-charter/issues/9
+URL: https://github.com/solid/solid-wg-charter/pull/38
 
+* SC: Resolves https://github.com/solid/solid-wg-charter/issues/9
+* SC: We have some agreement in the PR but there's a bit of paraphrasing we need to be careful with. I do want to wrap up the PR, but if we can try to integrate changes that would clarify. Any objections to merging this PR? 
+* ML: PAC wanted to answer but not sure if he did. 
+* SC: It is assigned to PAC. If we have agreement we would allow him to have last review and merge. 
+* SC: We can drop the word "new". We might have an exception with WebID. If it's listed as a deliverable it not intended to conflict with out of scope. 
+* SC: No objections.
 
-### Strategy for normative references
-URL: https://github.com/solid/solid-wg-charter/issues/37
+ACTION: SC to integrate change from discussion on PR with eP. PAC to have last review and merge.
 
 
 ### Consider adopting WebID 1.0 ED as a Deliverable
-https://github.com/solid/solid-wg-charter/issues/39
+URL: https://github.com/solid/solid-wg-charter/issues/39
+
+* SC: The proposal is to adopt WebID as a deliverable. We had several approvals from the CG perspective. At least one approval from WebID CG perspective. The specification is quite integral to Solid, been around for a long time, used by several communities outside Solid. It is mature. Things need to be improved, but it was deemed to be mature enough for solid ecosystem to use widely. Lot of our specs and implementations uses it. It is something we depend on. With respect to normative references, it would be in Solid WG's interest to run it through recommendation track. 
+* SC: We need agreement that we want to take it on and no objections from WebID CG.
+* JZ: Presumably this would ease the work of the Solid WebID-Profile group because we'd have something that is already on track to be accepted in the WebID spec itself. Is that correct?
+* SC: Yes.
+* JZ: Then I'm in favor. 
+* LD: This would mean we adopt the WebID spec within the WG? This would go beyond the WebID panel? Would also incorporate non-solid specific? 
+* SC: Yes. It's non-Solid specific. Solid WebID profile is one of the specs that uses it, other specs use it (AuthN, etc).
```

Comment:
```suggestion
* LD: This would mean we adopt the WebID spec within the WG? This would go beyond the WebID panel? Would also incorporate non-Solid-specific? 
* SC: Yes. It's non-Solid-specific. Solid WebID profile is one of the specs that uses it; other specs also use it (AuthN, etc.).
```